testing/mm/Kconfig

```diff
@@ -20,7 +20,7 @@ config TESTING_MM_PROGNAME
 
 config TESTING_MM_PRIORITY
 	int "Task priority"
-	default 100
+	default 250
```

Comment:
Why do you need to increase the priority? Does it fail when using priority 100?

Reply:
We run memtest in the background. The high prirority could avoid the bad high prority task block memtest.

Reply:
Ok! BTW it still be possible to have higher priority tasks (251-255)

Reply:
Yes, but it's rare:).